### ISSUE TITLE
[Spell] Kill Shot dipping below its minimum damage?

### ISSUE THREAD

**Issue by Aristo14** [WoWManiaUK/Redemption#2835]
http://www.wow-mania.com/armory/?spell=61006
http://www.wow-mania.com/armory/?item=92008

![screenshot_1](https://user-images.githubusercontent.com/38132718/47725668-3db0d800-dc59-11e8-982d-ef5a202fbb46.png)

On this screen you can see the calculation for Kill shot was properly made 
(rap 6743 * 0.40+352 * 2=3347,2)

![screenshot_2](https://user-images.githubusercontent.com/38132718/47725828-93858000-dc59-11e8-9288-c8c6d7adca88.png)

Showing I have the maniac's recurve equiped and with a 731-1003 weapon dmg this would result in:
731 * 200%+3347=4809  
1003 * 200%+3347=5353 

![screenshot_3](https://user-images.githubusercontent.com/38132718/47726088-0f7fc800-dc5a-11e8-9c74-08f105d5594a.png)

As you can see on this screen the damage dipped below the expected minimum even with gear and talent procs. 

The ? at the end of the github issue titel is because its tricky to test Kill Shot on normal toon, I did my shots in the mall on the crocs there.

**Comment by Hajzlopuc** [comment]
It's probably because the croc has high armor.

**Comment by Dranah** [comment]
Probably as Hajz says, your armor penetration can only remove majority of the targets armor, not all of it.
Most lvl 80 NPCs retain around 10-11% reduction with 100% armor penetration, those remaining armor require raid debuffs such as Sunder Armor and Faerie Fire to get off.

**Comment by Hajzlopuc** [comment]
Something to read 
http://wowwiki.wikia.com/wiki/Armor_penetration#Technical_explanation_from_Ghostcrawler

**Comment by Aristo14** [comment]
I am aware how arp works 

the crocs are lvl 78 and my hunter is arp capped. 
keep in mind i had 6.3 gear on so procs did happen. 

this issue doesnt have to be high prio but it does need testers to test it and not players (player will need his gear and therefore contaminate the test)

**Comment by Dranah** [comment]
It will be looked at.

**Comment by Aristo14** [comment]
thanks!

**Comment by Dranah** [comment]
Alright so, here is the detailed results:

1) Naked hunter with no talents
Weapon damage 241-261
Kill shot = 200% weapon damage + 786
I have 0 arpen which means 41,2% reduction applies from bosses.
So the calculated damage is 
MIN: (241x2+786)x0,588= 745 damage
MAX (261x2+786)x0,588= 769 damage
Tested damage on Lord Marrowgar, with ICC buff off : 746-768
Matches.

2) Geared hunter with no talents, no procs and no ARP. Same 41,2% armor reduction applies. Target is still Marrowgar.
Weapon damage 1646-2004
RAP = 4322
Calculated damage is
MIN: (1646x2+2378)x0,588 = 3333
MAX: (2004x2+2378)x0,588= 3754
Test damage = 3343-3752
Looks correct with gear scaling.

3) Now let's add talents to the hunter above. Still no procs, no ARP. Target is still Marrowgar.
Improved Tracking adds 5% extra damage towards tracked-type, in this case Undead.
Expected damage is now
MIN: 1,05 x 3333= 3499
MAX: 1,05x3754= 3941
Tested damage 3507-3933
Looks okay.
Ranged Weapon Specialization adds another 5% extra. Note that these talents currently stack additive cuz trinity.
Expected damage is now
MIN: 1,1 x 3333 = 3666
MAX: 1,1 x 3754 = 4129
Tested damage is 3682-4120
Looks correct.
Marked for Death adds another 5% when your target is marked.
Expected damage is now
MIN: 1,15 x 3333 = 3832
MAX: 1,15 x 3754 = 4317
Tested damage is 3678-4136
This does not look like it added any damage. Bummer, will give this an own thread soon.

4)Take off talents and add 100% arp. Still no procs and still against Marrowgar.
Hunter has now
4290 RAP
1904-2193 weapon damage
100% arp =9,2% damage reduction remains on target
Expected damage is now
MIN: (1904x2+2366)x0,908 = 5605
MAX: (2193x2+2366)x0,908 = 6130
Tested damage is 5602-6123
Looks fine.

5) Now let's add an armor debuff that takes off the remaining armor of target.
Tested with 5 stacks of sunder armor. Now the boss really has 0% mitigation.
Expected damage is now:
MIN: 1904x2+2366= 6174
MAX: 2193x2+2366= 6752
Tested damage = 6191-6745
Armor scaling is correct as well.

6) Now let's test with the same 100% Arp hunter on a Mall Croc.
Expected damage is same (or higher) than I measured on Lord Marrowgar, aka 5602-6123
Tested damage: 2785-2861.
Wow. These crocs are tough af.

Conclusion:
- Your calculations are off and you shouldn't be testing on the mall crocos.
- Marked for Death is not boosting Kill Shot damage, which means it will do 5% lower in a raid situation.
